lib/haversine.rb

```diff
@@ -12,7 +12,7 @@ def toRad
   end
 end
 
-class Fixnum
+class Integer
```

Comment:
It looks like this is true for Ruby 2.4 but not 2.3 . Am i correct?

Reply:
Yes, that looks to be correct, the deprivation warnings started after 2.4.0.

I've run tests against ruby 2.4.0 and 2.3.6 and it passes on both :).

Reply:
perfect. Thank you.

Reply:
Legend 👍, gimme a shout if this turns out to need further polish.

Oh, and happy new year!